README.md

```diff
@@ -1 +1,5 @@
 # Peer Programming Challanges
+Use this project as a way to challange developers with relativly real world examples related to RESTful APIs and frontend technologies. By loading this repository via [https://vscode.dev/github/heyitsgeo/peer-programming-challanges](https://vscode.dev/github/heyitsgeo/peer-programming-challanges) developers can edit code in a browser without needing to install or clone anything on their machine.
+
+## Required extensions
+* [vue](https://marketplace.visualstudio.com/items?itemName=jcbuisson.vue)
```

Comment:
What is this for? VS Code?

Reply:
Is this telling us that we need to install this or does the candidate need to install this? It's not clear to me what we are asking/instructing here.